WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderPostCardActionsHandler.kt

```diff
@@ -3,16 +3,16 @@ package org.wordpress.android.ui.reader.discover
 import android.content.ActivityNotFoundException
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MediatorLiveData
-import kotlinx.coroutines.CoroutineScope
```

Comment:
We are not launching new coroutines within the handler anymore. However, all the public methods are not suspendable and they move the execution to the bg thread.